### ISSUE TITLE
Source map not included in release

### ISSUE THREAD

**Issue by SirAiedail** [megawac/MutationObserver.js#29]
The source map is not included in the release download and the bower/npm packages.

**Comment by anonyco** [comment]
What is a source map?

**Comment by cs09g** [comment]
@anonyco see `//# sourceMappingURL=mutationobserver.map` at the end of the code

**Comment by tgroutars** [comment]
@megawac I have a warning when including this lib with parcel. Is it possible to include the source map in a new release?

**Comment by tgroutars** [comment]
Tried to build it myself, but I'm getting warnings
```
$ grunt build --force
Running "jshint:all" (jshint) task
>> 1 file lint free.

Running "connect:server" (connect) task
Started connect web server on http://0.0.0.0:8000

Running "qunit:main" (qunit) task
Testing http://localhost:8000/test/index.html
PhantomJS threw an error:ERROR
>> 0 [ '' ]
Warning: PhantomJS exited unexpectedly with exit code null. Used --force, continuing.
Warning: 0/0 assertions ran (0ms) Used --force, continuing.
(node:4429) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.

Running "closurecompiler:minify" (closurecompiler) task

Compiling MutationObserver.js -> dist/mutationobserver.min.js
>> MutationObserver.js:56: WARNING - The @expose annotation is deprecated. Use @nocollapse or @export instead.
>>     MutationObserver._period = 30 /*ms+runtime*/ ;
>>     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>>
>> MutationObserver.js:63: WARNING - The @expose annotation is deprecated. Use @nocollapse or @export instead.
>>     MutationObserver.prototype = {
>>     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>>
>> MutationObserver.js:72: WARNING - The @expose annotation is deprecated. Use @nocollapse or @export instead.
>>         observe: function($target, config) {
>>         ^^^^^^^
>>
>> MutationObserver.js:119: WARNING - Bad type annotation. Unknown type MutationRecord
>>          * @return {Array.<MutationRecord>}
>>                            ^
>>
>> MutationObserver.js:121: WARNING - The @expose annotation is deprecated. Use @nocollapse or @export instead.
>>         takeRecords: function() {
>>         ^^^^^^^^^^^
>>
>> MutationObserver.js:136: WARNING - The @expose annotation is deprecated. Use @nocollapse or @export instead.
>>         disconnect: function() {
>>         ^^^^^^^^^^
>>
>> MutationObserver.js:182: WARNING - Misplaced function annotation. This JSDoc is not attached to a function node. Are you missing parentheses?
>>         return function(mutations) {
>>         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>>
>> MutationObserver.js:186: WARNING - cannot instantiate non-constructor
>>                 mutations.push(new MutationRecord({
>>                                ^^^^^^^^^^^^^^^^^^^^
>>
>> MutationObserver.js:216: WARNING - assignment to property top of CSSStyleDeclaration
>> found   : number
>> required: string
>>     hasAttributeBug.style.top = 0;
>>     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>>
>> MutationObserver.js:217: WARNING - Property style never defined on NamedNodeMap<Attr>
>>     hasAttributeBug = hasAttributeBug.attributes.style.value != "null";
>>                                                  ^^^^^
>>
>> MutationObserver.js:230: WARNING - inconsistent return type
>> found   : string
>> required: (String|null)
>>         return attr.value;
>>                ^^^^^^^^^^
>>
>> MutationObserver.js:251: WARNING - Bad type annotation. Unknown type MutationRecord
>>      * @param {Array.<MutationRecord>} mutations
>>                       ^
>>
>> MutationObserver.js:266: WARNING - condition always evaluates to true
>> left : (String|null)
>> right: string
>>                 if (getAttributeValue($target, attr) !== $oldstate[name]) {
>>                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>>
>> 0 error(s), 13 warning(s), 75.6% typed
>> Complete: 4248

Running "qunit:min" (qunit) task
Testing http://localhost:8000/test/index.html?min=true
PhantomJS threw an error:ERROR
>> 0 [ '' ]
Warning: PhantomJS exited unexpectedly with exit code null. Used --force, continuing.
Warning: 0/0 assertions ran (0ms) Used --force, continuing.
(node:4429) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.

Running "file_info:source_files" (file_info) task

  Updating source_files stats in dist/README.md from:

  - Original: 25 kB
  - Minified: 3.7 kB
  - Gzipped:  1.6 kB


  to:

  - Original: 25 kB
  - Minified: 4.2 kB
  - Gzipped:  1.9 kB


  source_files stats:

  MutationObserver.js                25 kB (7.1 kB gzipped)
  dist/mutationobserver.min.js        4.2 kB (1.9 kB gzipped)

Done, but with warnings.
```

**Comment by drew-r** [comment]
```
WARNING in ./node_modules/mutationobserver-shim/dist/mutationobserver.min.js
(Emitted value instead of an instance of Error) Cannot find SourceMap 'mutationobserver.map': Error: Can't resolve './mutationobserver.map'
```

Please release with the source map.

**Comment by megawac** [comment]
I'll just remove the sourcemaps to kill the warning for now.

**Comment by MaicoTimmerman** [comment]
@megawac It seems the npm published 0.3.5 version for `dist/mutationobserver.min.js` is still the 0.3.3 version. So this issue is still present at this time.

**Comment by megawac** [comment]
Thanks for the heads up fixed it properly
